packages/sdks-tests/src/snippet-tests/editable-regions.spec.ts

```diff
@@ -22,27 +22,17 @@ test.describe('Editable regions in custom components', () => {
     page,
     packageName,
   }) => {
-    test.skip(!['react', 'angular', 'angular-ssr'].includes(packageName));
+    test.skip(
+      !['react', 'angular', 'angular-ssr', 'gen1-react', 'gen1-remix'].includes(packageName)
```

Comment:
this test is failing for `gen1-react`: https://github.com/BuilderIO/builder/actions/runs/12189792080/job/34005639784?pr=3750#step:5:1220

i see you added `'gen1-react'` here, but you didn't add any code to support it in this PR. is this a mistake?

Reply:
@samijaber Oh yeah, my mistake. Removing it.